packages/field-base/src/slot-target-mixin.js

```diff
@@ -38,28 +40,59 @@ const SlotTargetMixinImplementation = (superclass) =>
     }
 
     /**
-     * Forwards every node from the source slot to the target element
+     * Copies every node from the source slot to the target element
      * once the source slot' content is changed.
      *
      * @private
      */
-    __forwardNodesToSlotTarget() {
+    __checkAndCopyNodesToSlotTarget() {
+      this.__sourceSlotObserver.disconnect();
+
       if (!this._slotTarget) {
         return;
       }
 
+      // Remove any existing clones from the slot target
+      if (this.__slotTargetClones) {
+        this.__slotTargetClones.forEach((node) => this._slotTarget.removeChild(node));
+        delete this.__slotTargetClones;
+      }
+
       // Exclude whitespace text nodes
       const nodes = this._sourceSlot
         .assignedNodes({ flatten: true })
         .filter((node) => !(node.nodeType == Node.TEXT_NODE && node.textContent.trim() === ''));
 
       if (nodes.length > 0) {
-        this._slotTarget.replaceChildren(...nodes);
+        this.__copyNodesToSlotTarget(nodes);
       }
     }
+
+    /**
+     * Copies the nodes to the target element.
+     *
+     * @protected
+     * @param {Array<Node>} nodes
+     */
+    __copyNodesToSlotTarget(nodes) {
+      this.__slotTargetClones = this.__slotTargetClones || [];
+      nodes.forEach((node) => {
+        // Clone the nodes and append the clones to the target slot
```

Comment:
Does this also copy event listeners? I guess not, should we do anything about it?

Reply:
It doesn't, so it's a breaking change in that sense. The approach is similar to what we have in `<vaadin-select>`.

Both approaches have some kind of breaking change. We need to decide which one is the lesser evil.

Reply:
Sounds good to me.